python/datapointsfilters/covariancesampling.cpp

```diff
@@ -0,0 +1,31 @@
+#include "covariancesampling.h"
```

Comment:
Name of this file is not snake-cased, as promoted by Python.

Reply:
I wasn't sure which convention to use. I explained the reason behind this in my other comment. I'm fine using snake_case python convention.

Reply:
All files have been renamed according to the Python convention. So, I'll mark this as resolved. Thanks!